probe-rs/src/flashing/flasher.rs

```diff
@@ -641,7 +641,7 @@ impl<'probe, O: Operation> ActiveFlasher<'probe, O> {
             (regs.argument_register(3), registers.r3),
             (
                 regs.core_register(9),
-                if init {
+                if init && self.core.architecture() != Architecture::Xtensa {
```

Comment:
This seemed a rather ARM-specific write, I'm not sure what `static_base` is or where it's needed.

Reply:
Well, the whole CMSIS-DAP spec is architecture specific. It is issued by ARM.

Reply:
Hard to argue with that, but forgive me for getting confused when an ARM register name is used in an architecture-agnostic context :)